src/core/homepage_elements/about/hooks.py

```diff
@@ -3,9 +3,26 @@
 __license__ = "AGPL v3"
 __maintainer__ = "Birkbeck Centre for Technology and Publishing"
 
+from utils.setting_handler import get_plugin_setting
+from core.homepage_elements.about import plugin_settings
+
 
 def yield_homepage_element_context(request, homepage_elements):
     if homepage_elements is not None and homepage_elements.filter(name='About').exists():
-        return {'about_content': request.journal.description}
+
+        try:
+            title = get_plugin_setting(
+                plugin_settings.get_self(),
+                'about_title',
+                request.journal,
+            )
+            title_value = title.value
+        except IndexError:
+            title_value = 'About this Journal'
```

Comment:
Should we translate this? (the previous version was using a `{% trans %}` tag on the template

Reply:
Yes, I'll pull ugettext in and translate it.